ramalama/cli.py

```diff
@@ -1179,11 +1185,6 @@ def eprint(e, exit_code):
         eprint(f"pulling {e.geturl()} failed: {e}", errno.EINVAL)
     except HelpException:
         parser.print_help()
-    except AttributeError as e:
```

Comment:
I think we should remove this, I keep on doing it manually anyway, it keeps masking real issues with the misleading error:

"ramalama: requires a subcommand"

Reply:
Fine, I just use --debug to see the error, but removing it permanently, is a good idea, so if it happens in release we can get an easy bug report.